mtp_bank_admin/apps/bank_admin/statement.py

```diff
@@ -0,0 +1,117 @@
+import time
+import datetime
+
+from django.conf import settings
+from bai2 import bai2, models, constants
+from moj_auth import api_client
+
+from . import BAI2_STMT_LABEL
+from .exceptions import EmptyFileError
+from .utils import retrieve_all_transactions, get_transaction_uid,\
+    create_batch_record
+
+SENDER_ID = 'NWBKGB2L'
+RECEIVER_ID = '391796'
+ACCOUNT_NUMBER = '10002383 607080'
+CURRENCY = 'GBP'
+RECORD_LENGTH = 80
```

Comment:
Maybe? If we ever find ourselves with a test bank account or some such? Not too optimistic about that happening but sure, we can allow for it.